src/cockpit/channels/packages.py

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
meta: We recently started to prefix pybridge commits with "pybridge:". IMHO that makes more sense now, as we have plenty of other Python stuff in git, and python isn't a cockpit component.